resources/assets/sass/components/_tooltip.scss

```diff
@@ -0,0 +1,82 @@
+.c-tooltip {
+
+    @include color-modifiers-dark(color);
+
+    font-size: $font-size-small;
+    font-weight: $font-weight-medium;
+    position: relative;
+    display: block;
+    width: 100%;
+    padding: $spacer - 2 $spacer * 1.5 $spacer * 0.75;
+    background: $color-white;
+    border-radius: 3px;
+
+    &.m-one-line {
+
+        white-space: nowrap;
+    }
+
+    &:before {
+
+        content: '';
+        position: absolute;
+        width: 0;
+        height: 0;
+        border-style: solid;
+    }
+
+    &.m-top {
+
+        &:before {
+
+            top: -$spacer / 2;
+            left: $spacer;
+            border-width: 0 5.5px 6px 5.5px;
+            border-top-color: $transparent;
+            border-left-color: $transparent;
+            border-right-color: $transparent;
+            border-bottom-color: $color-white;
+        }
+    }
+
+    &.m-bottom {
+
+        &:before {
+
+            bottom: -$spacer / 2;
+            left: $spacer;
+            border-width: 6px 5.5px 0 5.5px;
+            border-top-color: $color-white;
+            border-left-color: $transparent;
+            border-right-color: $transparent;
+            border-bottom-color: $transparent;
+        }
+    }
+
+    &.m-inverted {
+
+        @include color-modifiers-dark-and(background);
+
+        color: $color-white;
+
+        &.m-top {
+
+            &:before {
+
+                border-bottom-color: $color-white;
+
+                @include color-modifiers-dark-and(border-bottom-color);
```

Comment:
Expected item on line 68 to appear before line 66. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets